### PR TITLE
Step/personal background check

### PR DESCRIPTION
**Pull request by queenieho**
This PR goes with this PR from blueprints: https://github.com/starcity-properties/blueprints/pull/32

image

image

### CODE REVIEW

File: resources/graphql/objects.edn [hunk 1]
```diff
@@ -43,45 +45,103 @@
...
+                              :description "Did the user give consent to perform a background check?"
+                              :resolve     (:get :community-safety/consent-given?)}
+                 :copy       {:type        Boolean
+                              :description "Does the user want a copy of this?"
+                              :resolve     (:get :community-safety/wants-report?)}
+                 :created    {:type    (non-null :Instant)
+                              :resolve :entity/created}}}
+
+
+  :background_check
```
Comment (allenjprice) [inline resources/graphql/objects.edn]: This made it in twice - please remove the second one!

File: src/cljs/apply/events.cljs [hunk 2]
```diff
@@ -145,23 +177,64 @@
  (fn [{db :db} [_ params]]
    ;; NOTE somehow graphql doesn't like communities being in a list
    ;; so I have to move it into a vector before the mutation
-   (let [params' (tb/transform-when-key-exists params
-                   {:communities #(into [] %)})]
-     (log/log "updating application " params')
+<<<<<<< HEAD
```
Comment (allenjprice) [inline src/cljs/apply/events.cljs]: Looks like there are some merge conflict artifacts in this namespace as well.

Comment (queenieho) [reply src/cljs/apply/events.cljs]: shoot, ill fix

**Comment by queenieho** [comment]
@allenjprice resolved issues

**Pull request merged**